### PR TITLE
MAT_077_H_implement_D2R_conversion

### PR DESCRIPTION
**Pull request by mircea-altair**
#### Description of the feature or the bug

   *MAT_077_H: implement D2R conversion

#### Description of the changes

    Add *CFG   for  *MAT_077_H: implement D2R conversion

### CODE REVIEW

File: hm_cfg_files/config/CFG/radioss2020/MAT/LAW95.cfg [hunk 1]
```diff
@@ -44,6 +44,8 @@ ATTRIBUTES(COMMON)
     SUBGRP_HEAT_MAT         = VALUE(SUBOBJECT, "");
     THERM_STRESS            = VALUE(INT, "Therm Stress");
     SUBGRP_THERM_STRESS     = VALUE(SUBOBJECT, "");
+    VISC_PRONY_option                       = VALUE(INT,"Visc Prony");
```
Comment (sebastienVilleneuve) [inline hm_cfg_files/config/CFG/radioss2020/MAT/LAW95.cfg:47]: maybe some incorrect indentation here ?

File: hm_cfg_files/messages/CONFIG/msg_hw_radioss_reader.txt [hunk 2]
```diff
@@ -279,4 +279,10 @@ duplicate %s id %d
 ** WARNING IN LS-DYNA IMPORT
 /MESSAGE/200027/DESCRIPTION
 %s (id: %u, name: %s) : First LCID function of *DEFINE_TABLE is used
-#------------------------------------------------------------------------------
+
+/MESSAGE/200028/TITLE
+** WARNING IN LS-DYNA IMPORT
+/MESSAGE/200028/DESCRIPTION
+%s (id: %u, name: %s) : The Mullins effect is not take into account
```
Comment (sebastienVilleneuve) [inline hm_cfg_files/messages/CONFIG/msg_hw_radioss_reader.txt:286]: I think there is a typo :

The Mullins effect is not take**n** into account

**Review by sebastienVilleneuve — CHANGES_REQUESTED** [review]
Hi @mircea-altair 

I think some small changes to clean the code .

Thanks,

File: hm_cfg_files/config/CFG/radioss2020/MAT/LAW95.cfg (hunk 1 above)
Comment (mircea-altair) [reply hm_cfg_files/config/CFG/radioss2020/MAT/LAW95.cfg:47]: Hi @sebastienVilleneuve ,

You are right. Please reject this pull request, and I'll do it again, after correction.
Thank you

File: hm_cfg_files/messages/CONFIG/msg_hw_radioss_reader.txt (hunk 2 above)
Comment (mircea-altair) [reply hm_cfg_files/messages/CONFIG/msg_hw_radioss_reader.txt:286]: What do you mean, please ?

**Comment by sebastienVilleneuve** [comment]
Hi @mircea-altair 

Can you correct , you have several commits in this PR ?

Thanks,
Sébastien

**Pull request closed without merging**